### PR TITLE
Initialize modalShown property of pageState to prevent race condition 

### PR DESCRIPTION
**Pull request by rtibbles**
## Summary

Revert's Object.assign in management pageState.

### CODE REVIEW

File: kolibri/plugins/management/assets/src/state/store.js [hunk 1]
```diff
@@ -41,7 +41,10 @@ const mutations = {
   SET_PAGE_STATE(state, pageState) {
     Object.assign(state.pageState, pageState);
   },
-
+  SET_CONTENT_PAGE_STATE(state, pageState) {
+    state.pageName = 'CONTENT_MGMT_PAGE';
+    state.pageState = pageState;
+  },
```
Comment (indirectlylit) [inline kolibri/plugins/management/assets/src/state/store.js]: Is this PR ready?

I see `SET_CONTENT_PAGE_STATE` added, but no references to it; also `SET_PAGE_STATE` still uses `Object.assign`

Comment (rtibbles) [reply kolibri/plugins/management/assets/src/state/store.js]: Ugh, it committed precisely the things I didn't want to. Will fix.

**Comment by rtibbles** [comment]
Should be the right commit now @indirectlylit

**Comment by indirectlylit** [comment]
Looks better :)

Did you confirm that the reactive array issue you'd mentioned didn't come back? I'm not sure where that manifested

**Comment by rtibbles** [comment]
Didn't see it recur (it was on content management, where newly imported channels were not being displayed).

**Pull request merged**